### PR TITLE
Test/stabilization8

### PR DESCRIPTION
**Pull request by clarissacastella**
Please review.

### CODE REVIEW

File: test/spec/app_spec.rb [hunk 1]
```diff
@@ -1066,21 +1066,98 @@
...
     end
 
-    # it "should find medias when searching by keyword" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should find medias when searching by keyword", bin2: true do
+      data = api_create_team_and_project
+      api_create_media(data: data, url: "https://www.facebook.com/permalink.php?story_fbid=10155901893214439&id=54421674438")
+      media = api_create_media(data: data, url: "https://twitter.com/TwitterVideo/status/931930009450795009")
+      @driver.navigate.to media.full_url
```
Comment (caiosba) [inline test/spec/app_spec.rb]: This step is useless, no?

File: test/spec/app_spec.rb [hunk 2]
```diff
@@ -1066,21 +1066,98 @@
...
+    end
+
+    it "should move media to another project", bin2: true do
+      data = api_create_team_and_project  
+      prj2 = api_create_project(data[:team].dbid.to_s)
+      p1 =  data[:project].team["projects"]["edges"][0]["node"]["title"]
+      p2 = prj2.team["projects"]["edges"][1]["node"]["title"]
+      source = api_create_media(data: data, url: "https://www.facebook.com/permalink.php?story_fbid=10155901893214439&id=54421674438")
+      @driver.navigate.to source.full_url
+      url1 = source.full_url
```
Comment (caiosba) [inline test/spec/app_spec.rb]: `source` here should be media, no?

Comment (clarissacastella) [reply test/spec/app_spec.rb]: Yes... It is just the var name. I'll change it.

File: test/spec/app_spec.rb (hunk 1 above)
Comment (clarissacastella) [reply test/spec/app_spec.rb]: It is there due the "get_team" in the next line.
get_team updated to data[:team].slug and step removed.

**Comment by clarissacastella** [comment]
Can I merge it in Dev?

**Pull request merged**